### ISSUE TITLE
Errors with Rails 5.0.2 - `NameError: uninitialized constant IRB`

### ISSUE THREAD

**Issue by ckib16** [mrkn/iruby-rails#1]
Hey there - 1st off thanks so much for writing this gem!

Unfortunately - I get these error when running in Jupyter after bundling & installing

![screen shot 2017-04-01 at 23 20 59](https://cloud.githubusercontent.com/assets/2160186/24584161/034de31c-1732-11e7-9ae4-b5b6ab45ebd5.png)

Here is the full text:
```ruby
Loading development environment (Rails 5.0.2)
NameError: uninitialized constant IRB
/Users/Chris/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/iruby-rails-1.0.0/lib/iruby/rails.rb:41:in `setup_console'
/Users/Chris/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/iruby-rails-1.0.0/lib/iruby/rails.rb:18:in `load'
/Users/Chris/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/iruby-rails-1.0.0/lib/iruby/rails.rb:60:in `load_rails'
<main>:in `<main>'
/Users/Chris/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/iruby-0.2.9/lib/iruby/backend.rb:44:in `eval'
/Users/Chris/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/iruby-0.2.9/lib/iruby/backend.rb:44:in `eval'
/Users/Chris/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/iruby-0.2.9/lib/iruby/backend.rb:12:in `eval'
/Users/Chris/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/iruby-0.2.9/lib/iruby/kernel.rb:87:in `execute_request'
/Users/Chris/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/iruby-0.2.9/lib/iruby/kernel.rb:47:in `dispatch'
/Users/Chris/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/iruby-0.2.9/lib/iruby/kernel.rb:37:in `run'
/Users/Chris/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/iruby-0.2.9/lib/iruby/command.rb:70:in `run_kernel'
/Users/Chris/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/iruby-0.2.9/lib/iruby/command.rb:34:in `run'
/Users/Chris/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/iruby-0.2.9/bin/iruby:5:in `<top (required)>'
/Users/Chris/.rbenv/versions/2.4.1/bin/iruby:22:in `load'
/Users/Chris/.rbenv/versions/2.4.1/bin/iruby:22:in `<main>'
```

**Comment by genya0407** [comment]
I also encountered the same error.

I don't understand why it occurs, but solution is:

`Gemfile`:

```ruby
gem 'pry', group: :development
```

`config/environments/development.rb` :

```ruby
Rails.application.configure do
  ...
  console do
    # this block is called only when running console,
    # so we can safely require pry here
    require "pry"
    config.console = Pry
  end
  ...
end
```

**Comment by genya0407** [comment]
Another solution is require irb manually.

In your jupyter notebook's cell:

```ruby
require 'iruby/rails'
require 'irb'
IRuby.load_rails
```

This will work fine.

**Comment by mrkn** [comment]
@ckib16 Please add `gem 'pry', group: :development` in the Gemfile of your Rails.
It may fix the problem.

**Comment by genya0407** [comment]
If iruby uses pry as backend, the same error occurs.

PlainBackend(irb):

![plain_backend](https://user-images.githubusercontent.com/3054386/27181874-65878322-5214-11e7-8602-a720a2a14b3b.png)

PryBackend(pry):
![pry_backend](https://user-images.githubusercontent.com/3054386/27181875-65929ece-5214-11e7-97c3-b177b12b6e99.png)

**Comment by mrkn** [comment]
It should be fixed by #3
